llzk-sys/src/build_support/mod.rs

```diff
@@ -35,12 +37,33 @@ fn build_llzk_inner(default_cfg: &DefaultConfig) -> Result<LlzkBuild> {
 }
 
 pub fn build_llzk(default_cfg: &DefaultConfig) -> Result<LlzkBuild> {
-    let mut llzk = build_llzk_inner(default_cfg)?;
-    for path in llzk.link_paths()? {
-        println!("cargo:rustc-link-search={}", path.display());
-    }
-    for lib in llzk.library_names()? {
-        println!("cargo:rustc-link-lib={lib}");
+    let llzk = build_llzk_inner(default_cfg)?;
+    // TODO: Change the path `llzk` keeps track of to be $OUT instead of the current: $OUT/build
+    // Since the libs are actually installed in $OUT/$LIBDIR.
```

Comment:
is this addressed in one of the other PR's or does it need to be addressed here?